src/users/dtos/CreateUser.dto.ts

```diff
@@ -14,4 +14,8 @@ export class CreateUserDto {
   @ApiProperty()
   @IsNotEmpty()
   age: number;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  role: string;
```

Comment:
Same here. It's better if we go with enums

Reply:
lemme work on that thanks